common/utils.js

```diff
@@ -454,7 +454,9 @@ const getItsEdgeContract = (chainConfig) => {
 };
 
 const getItsEdgeChains = (config, excludeChainName) => {
```

Comment:
```suggestion
const itsEdgeChains = (config, excludeChainName) => {
```
should avoid `get` prefix